app/js/arethusa.search/services/search.js

```diff
@@ -288,11 +289,39 @@ angular.module('arethusa.search').service('search', [
 
 
     function compareWords(wordA,wordB) {
-      // todo we may want to support additional language
-      // specific normalization
-      return wordA === wordB;
+      return normalize(wordA) === normalize(wordB);
     };
 
+    function normalize (text) {
+      if (text && self.language) {
+        // These normalizations functions come from the Alpheios core
+        // language models (https://github.com/alpheios-project/alpheios-core)
+        if(self.language.lang === 'gr') {
+          // we normalize greek to NFC - Normalization Form Canonical Composition
+          text = text.normalize('NFC');
```

Comment:
I would consider normalizing regardless of language (unless there are other issues with doing that). Even in a treebank of English text, there could be problems with non-normalized Unicode in words like resumé: the é could be either `\u00E9` or `\u0065\u0301`.

Maybe the code could look something like:

```javascript
function normalize (text) {
  if (text) {
    text = text.normalize('NFC');

    if (self.language.lang === 'grc') {
      // Greek-specific stuff
    }
    if (self.language.lang === 'lat') {
      // Latin-specific stuff
    }
  }
```

Reply:
that's a good suggestion. I think it should be fine to do, although I'm not positive if it has an undesirable side effects in languages other than Greek, but it seems unlikely. I will go ahead and make that change.